### DISCUSSION TITLE
Auto brightness and auto color_temperatur

### DISCUSSION THREAD

**Discussion by EbbiKS** [andesse/hue-clip-api.node-red-flows#5] [Q&A] [answered]
Thanks for all the work you put into the API flow. I was able to replace almost all of my magichue nodes.  Which finally made my setup usable again. 

There is one feature of Hue Magic that I am currently missing. There was the possibility to set brightness and temperature depending on the time of day, just by setting brightness to "auto" and color_temperature to "auto". The value "auto" is apparently not accepted by the API. Have you maybe already found a solution for this as well?

**Comment by andesse** [comment, marked as answer]
Hi @EbbiKS i found following in the api. Maybe the complete feature you are asking for isn’t available yet. Hue still adds features to v2. You could try this, I am not at home for testing until Wednesday. 

timed_effects: (object)
Basic feature containing timed effect properties.

effect: (one of sunrise, no_effect)
duration: (integer)
Duration is mandatory when timed effect is set except for no_effect. Resolution decreases for a larger duration. e.g Effects with duration smaller than a minute will be rounded to a resolution of 1s, while effects with duration larger than an hour will be arounded up to a resolution of 300s. Duration has a max of 21600000 ms.

**Comment by GeorgeCastanza** [comment]
Did you figure this out @EbbiKS ??

I'd love auto for brightness and temperature, or a workaround of some description.

**Reply by andesse** [reply]
@GeorgeCastanza 
This might not yet in the api V2. 

idea/example:
I have a wall switch for a room and use  „within time nodes“ to trigger 2 scenes during the day (day/evening)
![IMG_2591](https://user-images.githubusercontent.com/76150626/186593631-fb8675b3-87da-4090-b40a-5bdc41ae5ffd.jpeg)

You could use these nodes too (probably even more of them) to trigger different scenes during the day. 

Greetings 
Andreas
